### ISSUE TITLE
Can I use a computed property based on a local storage key?

### ISSUE THREAD

**Issue by jrdn91** [mfeckie/ember-cli-local-storage-wrapper#1]
I had something setup before using a different local storage method but I am switching over to using your plugin. I have this computed property

```
winning: Ember.computed('item.item_high_bid_user_id','localUser.user.id',function(){
    return this.get('item.item_high_bid_user_id') == this.get('localUser.user.id');
}),
```

Can I create computeds from your service?

**Comment by mfeckie** [comment]
Hmm, it's not something I've needed to do.  I suspect it would be challenging because there is no event you could hook into at the JS level to notify if localStorage items change.

There's a bit more info on why, here http://stackoverflow.com/questions/4671852/how-to-bind-to-localstorage-change-event-using-jquery-for-all-browsers

I may have misunderstood your question thought, so please correct me if that's the case :)

**Comment by jrdn91** [comment]
So not sure how this could potentially be adapted into the addon, but here is what I ended up doing.

I pulled your service code and essentially added an init function

```
init: function() {
    this.set('user',this.getItem('user'));
},
```

Which is also defined as a property on the service as just `user: null`, this way when the service is initialized it always gets what's in the local storage for user and has it set at the beginning. This means I had to initialize it instead of injecting it into my routes so that this happened first before the routes or wherever I wanted it had the data available to them.

The last piece was to update the `user` property on the service when the `setItem` method is called so that anything being computed or observing this property would notice the change, that is done like so in the `setItem` method

```
this.set(key,object);
```

This is pretty clean and simple but could use refinement I know because it requires me to know at the outset what properties I will need for my app to watch for.

Maybe when I have more time I'll revisit this and attempt to make this more autonomous. But just thought I would put it here as food for thought.
